pkgs/development/libraries/libvmi/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv,
+  fetchFromGitHub,
+  which,
+  autoreconfHook,
+  autoconf,
+  automake,
+  libtool,
+  yacc,
+  bison,
+  flex,
+  glib,
+  pkgconfig,
+  json_c,
+  xen,
+  libvirt,
+  xenSupport ? true }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "libvmi-${version}";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "libvmi";
+    repo = "libvmi";
+    rev = "v${version}";
+    sha256 = "0wbi2nasb1gbci6cq23g6kq7i10rwi1y7r44rl03icr5prqjpdyv";
+  };
+
+  buildInputs = [ glib which libvirt json_c ] ++ (optional xenSupport xen);
+  nativeBuildInputs = [ autoreconfHook yacc bison flex libtool autoconf automake pkgconfig ];
```

Comment:
`yacc`, `autoconf` and `automake` aren't needed.

Reply:
Yes, you're right. I didn't question the inputs from @proteansec. Only one of either `yacc` or `bison` is required. I'll remove `yacc`.

Reply:
I guess `autoconf` and `automake` are implicitly marked as `nativeBuildInputs` by `autoreconfHook`?